pkg/remote/aws/error.go

```diff
@@ -0,0 +1,61 @@
+package aws
+
+import "fmt"
+
+type ListError interface {
```

Comment:
`ListError` sound too much generic imho, what about something like `ResourceEnumerationError` ?

Reply:
:+1:

Reply:
I'd rather like the most verbose one